Sources/StreamChatUI/Components.swift

```diff
@@ -198,8 +198,21 @@ public struct _Components<ExtraData: ExtraDataTypes> {
     public var currentUserAvatarView: _CurrentChatUserAvatarView<ExtraData>.Type =
         _CurrentChatUserAvatarView<ExtraData>.self
 
+    // MARK: - Navigation
+
+    /// The navigation bar.
+    public var navigationBar: ChatNavigationBar.Type = ChatNavigationBar.self
```

Comment:
This component is not used by default in UI SDK and customers have to deal with it creating the navigation controller with our navigation bar on their side (the code-snippet is taken from `Sample` app):

```swift 
let navigationController = UINavigationController(
 navigationBarClass: channelList.components.navigationBar.self,
 toolbarClass: nil
)
```
We might consider having a navigation controller subclass with the `components.navigationBar` set.

Reply:
Yep, I like the idea of exposing this as a `UINavigationController` subclass 👍

Reply:
✅ Introduced in 78bb645e1ffea4adb4d80f0892ab03fc24ea50f8